src/DynamoCore/Library/DocumentationServices.cs

```diff
@@ -76,14 +76,33 @@ public static XmlReader GetForAssembly(string assemblyPath, IPathManager pathMan
             if (File.Exists(documentationPath))
                 return true;
 
+            //try with uppercase XML for linux
+            documentationPath = PathHelper.replaceExtension(documentationPath, ".XML");
```

Comment:
I was thinking something similar.  There could be hard to trace issues if we end up with multiple files that differ only in case.